globus_automate_client/cli/flows.py

```diff
@@ -507,14 +516,22 @@ def flow_run(
         case_sensitive=False,
         show_default=True,
     ),
+    dry_run: bool = typer.Option(
+        False,
+        "--dry-run",
+        help=(
+            "Do a dry run with your input to this flow to test the input without"
+            " actually running anything."
```

Comment:
`dry_run` in the case of running a Flow checks validation against a Flow schema? Or is it checking more than that? Not sure what creating an `Action` on the Flow's side is validating.